packages/button-react/README.md

```diff
@@ -7,3 +7,43 @@
 ## Kom i gang
 
 [Lær hvordan du kan ta i bruk Jøkul](https://fremtind.github.io/jokul/developer/getting-started/)
+
+## Bruk av React-pakken
+
+### Installasjon
+
+1. Installér pakken med `yarn add @fremtind/jkl-button-react` eller `npm i @fremtind/jkl-button-react`. Stil-pakken blir automatisk installert som en avhengighet.
+
+2. Importer _både_ React-pakken (velg komponenten(e) du trenger) og stil-pakken i prosjektet ditt:
+
+```js
+import { PrimaryButton, SecondaryButton, TertiaryButton } from "@fremtind/jkl-button-react";
+import "@fremtind/jkl-button/button.min.css";
+```
+
+### Bruk
+
+Komponentene tar følgende props:
+
+-   `children`: **Påkrevd**. Innholdet i knappen. `ReactNode`
+-   `onClick`: **Påkrevd**. Klikkhåndtering for knappen. Får en MouseEvent som første argument. `MouseEventHandler<HTMLButtonElement>`
+-   `forceCompact`: Angir at knappen skal vises i liten versjon uavhengig av skjermstørrelse. `boolean`
+-   `className`: Eventuell(e) css-klassenavn for komponenten. `string`
```

Comment:
Som i en annen PR, jeg er redd for at denne fort havner i usync med koden, og gjør mer vondt enn godt. Det er stort sett dekket av index.d.ts fila.

Reply:
Ser bekymringen. Alternativer blilr å ha attributtnavn som helt utvedtydig hører til en variant/et bruksområde av komponenten. I de fleste komponentene er ikke det noe problem, men jeg kan se for meg et par komponenter der det vil være greit med en ekstra forklaring..